app/services/sessions.js

```diff
@@ -25,3 +36,18 @@ exports.deleteAll = e => {
     throw errors.databaseError(err.detail);
   });
 };
+
+exports.getOne = (e, t) => {
+  return orm.models.sessions.findOne({ where: { email: t, token: t } }).catch(err => {
```

Comment:
why are you using `t` on both where conditions?
Also, change the parameter names to email and token so that you can do `{ where: { email, token } }`